data/Ousman37.json

```
@@ -1,17 +1,86 @@
 {
   "name": "Ousman Ethmane",
   "bio": "Passionate Front-End Developer | UI/UX Enthusiast | Building innovative web experiences",
+  "tags": [
+    "Open Source",
+    "HTML",
+    "CSS",
+    "JavaScript",
+    "React.js",
+    "Node.js",
+    "Git",
+    "GitHub",
+    "TypeScript"
+  ],
+  "evenTags": [
+    "Front-End Development",
+    "UI/UX Design",
+    "Responsive Web Design",
+    "Version Control",
+    "Web Development Tools",
+    "Problem Solving",
+    "Debugging",
+    "Team Collaboration"
+  ],
+  "socials": [
+    {
+      "name": "LinkedIn Follow me",
+      "url": "https://www.linkedin.com/in/ethmane-ousman-34885844/",
+      "icon": "fab fa-linkedin",
```

This is not a valid icon and I have changed this to what I believe you are looking to achieve:
```suggestion
      "icon": "FaLinkedin",
```

---

Thank you, @SaraJaoude  for your feedback and suggestion. I appreciate your input. I've made the necessary changes to use the custom icon format 'FaLinkedin' as you suggested. Please let me know if there are any further improvements or adjustments needed.